qutip/core/cy/qobjevo.pyx

```diff
@@ -187,13 +198,18 @@ cdef class QobjEvo:
         qevo = H0 + H1 * coeff
 
     """
-    def __init__(QobjEvo self, Q_object, args=None, tlist=None,
-                 order=3, copy=True, compress=True,
-                 function_style=None, boundary_conditions=None):
+    def __init__(QobjEvo self, Q_object, args=None, copy=True, compress=True, *,
```

Comment:
```suggestion
    def __init__(QobjEvo self, Q_object, args=None, *, copy=True, compress=True,
```
If we're going to make some arguments keyword-only, perhaps these should include `copy` and `compress`? The `Q_object` and `args` are the two really common ones.